documents/extension/funding/funding.go

```diff
@@ -0,0 +1,33 @@
+package funding
+
+import (
+	"reflect"
+
+	clientfundingpb "github.com/centrifuge/go-centrifuge/protobufs/gen/go/funding"
+)
+
+// Data is the default funding extension schema
+type Data struct {
+	FundingId             string `json:"funding_id,omitempty" attr:"string"`
+	Amount                string `json:"amount,omitempty" attr:"string"`
+	Apr                   string `json:"apr,omitempty" attr:"string"`
+	Days                  string `json:"days,omitempty" attr:"integer"`
+	Fee                   string `json:"fee,omitempty" attr:"string"`
```

Comment:
apr and fee should also be timestamp

Reply:
I think both should be `decimal` :)

apr ->  annual percentage rate
fee -> a value in dollar or ETH

Reply:
oops! my bad